geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/TiledRasterRDD.scala

```diff
@@ -66,7 +66,7 @@ class SpatialTiledRasterRDD(
 }
 
 
-class SpatialTemporalTiledRasterRDD(
+class TemporalTiledRasterRDD(
```

Comment:
Yeah, I buy this, we're already diverging from GT API a little and having a shorter names makes this a little easier to think about.